### ISSUE TITLE
Error when dynamically changing microphone device (Firefox only)

### ISSUE THREAD

**Issue by mariogasparoni** [bigbluebutton/bigbluebutton#12023]
**Describe the bug**
We can't dynamically change microphone in firefox browser. The browser console also outputs this error
```
Uncaught (in promise) DOMException: Concurrent mic process limit.
```

**To Reproduce**
1. Join full audio using firefox browser
2. Click on microphone and select any other microphone device.
3. Change won't take effect

**Expected behavior**
Microphone device should be changed

**Actual behavior**
Microphone device is not being changed

**Screenshots**
![error](https://user-images.githubusercontent.com/1780868/114723706-3f7b9280-9d11-11eb-8b1f-c4bb183009b0.png)

**BBB version (optional):**
2.3-beta-3

**Browser info**
Firefox 87.0 (64-bits) , linux version

Thanks to @pedrobmarin  , who found this one.

**Comment by mariogasparoni** [comment]
If you set the following value in bbb-html5's `settings.yml`:
```
...
mutedAlertenabled:
  enabled: false
...
```
the problem won't happen.

**Comment by PhMemmel** [comment]
I was just about to open a separate issue, but probably it's somehow the same.

Biggest difference with my Firefox: No error logs in console, everything looks fine. Audio devices aren't switched properly though despite UI suggesting they are.

BTW: When trying to switch FF re-requests permission for the correct selected device. Audio input doesn't change though, still remains on first connected device.

Switching audio input channel via echo-test-red-thumbs-down-modal works.

I tried to disable mutedAlert like you wrote and you are right: problem disappears. So I decided to comment on this issue instead of opening a new one.
